python/tests/api/store/test_sqlite_store.py

```diff
@@ -0,0 +1,55 @@
+import sqlite3
+from datetime import datetime, timedelta
+
+import pytest
+
+from whylogs.api.store.query import DateQuery, ProfileNameQuery
+from whylogs.api.store.sqlite_store import SQLiteStore
+from whylogs.core import DatasetProfileView
+
+
+@pytest.fixture
+def connection():
+    connection = sqlite3.connect(":memory:")
+    return connection
+
+
+@pytest.fixture
+def store(profile_view, connection):
+    store = SQLiteStore(connection=connection)
+    store.write(profile_view=profile_view, profile_name="my_profile")
+    store.write(profile_view=profile_view, profile_name="my_profile")
+    yield store
+    del store
```

Comment:
might be worth considering the with pattern. Isn't that how people do resource auto closing in Python?

Reply:
accepts both ways now :)